tv_apps/browser/js/browser_dialog.js

```diff
@@ -222,6 +244,16 @@ var BrowserDialog = {
         });
         break;
 
+      case 'sync_error':
+        this.openAlertDialog({
+          titleL10nId: options.titleL10nId,
+          messageL10nId: options.messageL10nId,
```

Comment:
The fx-sync error l10n id only support 3 type of errors: ERROR_UNKNOWN, ERROR_INVALID_SYNC_ACCOUNT and ERROR_OFFLINE.
And these three l10n id are all support in translation file.

Please see the tv_apps/browser/js/sync/settings.js, line 234-244
```
var errorIds = [ERROR_UNKNOWN, ERROR_UNKNOWN + '-explanation'];

const KNOWN_ERRORS = [
    ERROR_INVALID_SYNC_ACCOUNT,
    ERROR_OFFLINE
];
```